modules/nw-egress-ips-about.adoc

```diff
@@ -164,7 +164,7 @@ ifdef::ovn[]
 [id="nw-egress-ips-multi-nic-considerations_{context}"]
 == Considerations for using an egress IP on additional network interfaces
 
-In {product-title}, egress IPs provide administrators a way to control network traffic. Egress IPs can be used with the `br-ex`, or primary, network interface, which is a Linux bridge interface associated with Open vSwitch, or they can be used with additional network interfaces.
+In {product-title}, egress IPs provide administrators a way to control network traffic. Egress IPs can be used with a `br-ex` Open vSwitch (OVS) bridge, a primary network interface, or additional network interfaces.
```

Comment:
This seems a little awkward to me. Can we just say something like:

Egress IPs can be used with any physical interface with IP connectivity, as well as the br-ex Open vSwitch (OVS) bridge interface.

Reply:
Thanks, @trozet . I updated the statement as suggestion. Please leave a "LGTM" if all looks OK.